custom-commands/platformsh-exporting/README.md

```diff
@@ -0,0 +1,9 @@
+# Downloading data from Platform.sh into DDEV
+
+To download your site’s data for local development:
+
+1. Copy the [sync](./sync) command to the `.ddev/commands/host/` directory.
+2. Set the `PLATFORM_PROJECT` variable to the ID of your project.
+3. Run `ddev sync` to download data.
```

Comment:
Since this is very platform-sh-specific, I think it would be better for the command to have a name reflecting that. Also, it's one-directional right? So "sync" might not be the right name anyway.

Reply:
Done